images/reviewer_bw2/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+ARG VERSION=${VERSION:-0.2.7-bw2-fork-0.2}
+# Install Git to clone the repository
+RUN apt-get update && \
+    apt-get install -y \
+    cmake \
+    g++ \
+    git \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    zlib1g-dev
+# Clone the repository, jump into it
+# build the REViewer executable, +x, move to /usr/bin
+RUN git clone https://github.com/bw2/REViewer && \
+    cd REViewer && \
+    mkdir build && \
+    cd build && \
+    cmake .. && \
+    make && \
+    chmod +x /REViewer/build/install/bin/REViewer && \
+    mv /REViewer/build/install/bin/REViewer /usr/bin
```

```suggestion
    mv /REViewer/build/install/bin/* /usr/bin && \
    cd . && \
    rm -rf REViewer
```

I Just had a look at the image size of my local build - 1.8GB, with 1.5 GB taken up by the REViewer build resources. With the compiled executable I think all that can be deleted